frontend/package.json

```diff
@@ -75,7 +75,6 @@
     ]
   },
   "devDependencies": {
-    "@graphql-codegen/cli": "2.8.1",
```

Comment:
Cleanup: when we run the `yarn run codegen` script, it looks for the `@graphql-codegen/cli` package in the root directory, but this package is currently installed in the `frontend` directory. I'm moving this package to the root directory.